base/applications/cmdutils/replace/lang/sq-AL.rc

```diff
@@ -0,0 +1,52 @@
+/* TRANSLATOR : Ardit Dani (Ard1t) (ardit.dani@gmail.com)
+ * DATE OF TR:  29-11-2013
+*/
+
+LANGUAGE LANG_ALBANIAN, SUBLANG_NEUTRAL
+
+STRINGTABLE
+BEGIN
+    STRING_CONSOLE_ERROR "Error i paditur: %d\n"
+    STRING_REPLACE_HELP1 "zëvendëson dokumentat.\n\n\
+REPLACE [drive1:][rrugë1]emri [drive2:][rrugë2] [/A] [/P] [/R] [/W]\n\
+REPLACE [drive1:][rrugë1]emri [drive2:][rrugë2] [/P] [/R] [/S] [/W] [/U]\n\n\
+  [drive1:][rrugë1]emri Specifikon e dokumentit ose dokumentëve.\n\
+  [drive2:][rrugë2]         Specifikon directorine ku dokumentet do të\n\
+                           zëvendësohen.\n\
+  /A                       Shton dokumente të reja tek directoria e destinacionit. Nuk\n\
+                           përdoret me /S ose /U.\n\
+  /P                       Shfaw konfirmimim para se të zëvendësosh një dokument apo\n\
+                           shtimit të burimit të dokumentit.\n\
+  /R                       Zëvendëso dokumentat vetëm-lexohet, si dhe dokumentet e\n\
+                           pambrojtur.\n\
+  /S                       Zëvendëson dokumentet në të gjitha nën-direktoritë e\n\
+                           directorite. Nuk mund të përdorë me/A\n\
+                           parametra.\n\
+  /W                       Pret për ju për të futur një disk para fillimit të.\n\
+  /U                       Zëvendëso (update) vetem dokumenta që janë më të vjetër se\n\
+                           dokumentat e burimit. Nuk mund ta përdorësh me /A.\n"
+    STRING_REPLACE_HELP2 "Rruga e Burimi kërkohet\n"
+    STRING_REPLACE_HELP3 "Nuk zëvendësohen dokumentat\n"
+    STRING_REPLACE_HELP4 "%lu dokumenta(t) zëvendësohen\n"
+    STRING_REPLACE_HELP5 "Zëvendësohen %s\n"
+    STRING_REPLACE_HELP6 "Zëvendësoh %s\n"
+    STRING_REPLACE_HELP7 "S'ka dokumentat te shtuar\n"
+    STRING_REPLACE_HELP8 "%lu dokumenta(t) u shtuan\n"
+    STRING_REPLACE_HELP9 "Shto %s (P/J)? "
+    STRING_REPLACE_HELP10 "Zëvendëso %s (P/J)? "
+    STRING_REPLACE_HELP11 "Shtim %s\n"
+
+    STRING_REPLACE_ERROR1 "Qeles invalid - %s\n"
+    STRING_REPLACE_ERROR2 "Rruga nuk u gjet - %s\n"
+    STRING_REPLACE_ERROR3 "Dokumenti, emri i direktorise, ose syntaksi i etiketes volumetrike eshte gabim.\n"
+    STRING_REPLACE_ERROR4 "Kombimi i parametrave invalid\n"
+    STRING_REPLACE_ERROR5 "Refuzot hyrja - %s\n"
+    STRING_REPLACE_ERROR6 "Nuk u gjet dokumenti - %s\n"
+    STRING_REPLACE_ERROR7 "Error i zgjatur 32\n"
+
+    STRING_COPY_OPTION "YNA"
```

Comment:
These should be changed to
```suggestion
    STRING_COPY_OPTION "PJT"
```
(corresponds to: "Yes, No, All" --> "Po, Jo, Te" in albanese)
here, and also in the cmd sq-AL.rc translation.
Because the replace-questions (see above) show the choice "P/J" ("Yes/No"), pointing the user to enter either of these, but these wouldn't be recognized because the copy-option mismatches.

Reply:
Fixed.